src/components/Textfields/TextFieldBase/TextfieldBase.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import { makeStyles, Theme } from "@material-ui/core";
+
+import TextfieldBase, { TexfieldBaseProps } from ".";
+
+import { Story, Meta, StoryContext } from "@storybook/react";
+
+const useStyles = makeStyles<Theme, StoryContext>((theme) => ({
+    root: {
+        backgroundColor: (context) =>
+            context.args.color === "secondary"
+                ? theme.palette.common.black
+                : theme.palette.common.white,
```

Comment:
Please understand that the color of material UI is based around _Surface_ and _On Surface_
![image](https://user-images.githubusercontent.com/26225039/116805916-de96eb80-ab53-11eb-8081-1953c2c2b2b9.png)
Please use a background color and the color that correspond to that background instead

Reply:
For example, if the background is background A, the color should be On A.

Reply:
On A in the code is represented by the contrastText property